client/src/Movies/SavedList.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
+import {NavLink} from 'react-router-dom';
+
 
 const SavedList = props => (
+  
   <div className="saved-list">
     <h3>Saved Movies:</h3>
     {props.list.map(movie => (
       <span className="saved-movie">{movie.title}</span>
     ))}
-    <div className="home-button">Home</div>
+    <NavLink to="/" >
+      <div className="home-button">Home</div>
+    </NavLink>
```

Comment:
Good use of the built-in NavLink to get the provided aria focus attributes and styles.